src/renderer/views/LobbyView/index.tsx

```diff
@@ -33,15 +24,15 @@ function LobbyView() {
 
   useEffect(() => {
     if (error) history.push("/error/relaunch");
-  }, [error]);
+  }, [history, error]);
 
   return (
     <>
       <Layout />
       <OnboardingOverlay
         // isOpen={onboardingRequired && showOnboarding}
         isOpen={false}
-        onClose={() => setShowOnboarding(false)}
+        onClose={() => {}}
```

Comment:
Is this handler still needed?

Reply:
I don't think so, but the `onClose` prop was a required prop.

Reply:
Then... how about loosening `OnboardingOverlay.onClose`'s requirement as optional?

Reply:
I'll check and loose the restraint. Thanks for your review!

Reply:
Yup, that OnboardingOverlay supposed to be a welcome message to newly activated user, but never actually used properly. 

Since we now have that as a subview in this PR, I can safely confirm you can remove that.👌